src/containerbuddy/config.go

```diff
@@ -20,8 +20,12 @@ var (
 type Config struct {
 	Consul       string `json:"consul,omitempty"`
 	OnStart      string `json:"onStart"`
+	PreStop      string `json:"preStop"`
+	PostStop     string `json:"postStop"`
 	StopTimeout  int    `json:"stopTimeout"`
 	onStartArgs  []string
+	preStopArgs  []string
+	postStopArgs []string
```

Comment:
I did a search thru the code and it looks like `onStartArgs`, `preStopArgs`, and `postStopArgs` are never used. You're passing the equivalent public member of the struct to `runCommand` instead, so there's no reason to have these.